crates/uv-settings/src/lib.rs

```diff
@@ -57,22 +58,25 @@ impl FilesystemOptions {
 
     /// Find the [`FilesystemOptions`] for the given path.
     ///
-    /// The search starts at the given path and goes up the directory tree until a `uv.toml` file is
-    /// found.
+    /// The search starts at the given path and goes up the directory tree until a `uv.toml` file or
+    /// `pyproject.toml` file is found.
     pub fn find(path: impl AsRef<Path>) -> Result<Option<Self>, Error> {
         for ancestor in path.as_ref().ancestors() {
-            // Read a `uv.toml` file in the current directory.
-            let path = ancestor.join("uv.toml");
-            match fs_err::read_to_string(&path) {
-                Ok(content) => {
-                    let options: Options = toml::from_str(&content)
-                        .map_err(|err| Error::UvToml(path.user_display().to_string(), err))?;
-
-                    debug!("Found workspace configuration at `{}`", path.display());
-                    return Ok(Some(Self(options)));
```

Comment:
Did you mean to drop this? Does this happen elsewhere now somehow?

Reply:
Yeah this is in `from_directory`.